### PR TITLE
Release 66

### PR DESCRIPTION
**Pull request by erkangz**

### CODE REVIEW

File: release-notes/2021-66.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+  - Are there any custom role definitions with assignableScope of subscription
+    or higher and action of *? (1.23/1.21)
+  - Is Azure Security Defaults enabled for all Microsoft directories/tenants?
+    (1.22)
+  - Are there any Azure storage accounts that have not enabled Secure Transfer?
+    (3.1)
+  - Are all Azure storage accounts configured with "Deny" as the default network
+    access rule? (3.7/3.6)
+  - Are "Trusted Microsoft Services" enabled for Azure storage accounts?
+    (3.8/3.7)
```
Comment (ndowmon) [inline release-notes/2021-66.md:45]: See https://bitbucket.org/jupiterone/jupiter-integration-azure/pull-requests/132/add-azure-cis-benchmark-36-39

```suggestion
  - Are all Azure storage accounts configured with "Deny" as the default network
    access rule? (3.7/3.6)
  - Are "Trusted Microsoft Services" enabled for Azure storage accounts?
    (3.8/3.7)
  - Is "Soft Delete" enabled for Azure storage accounts?
    (na/3.8)
  - Are Azure storage accounts encrypted with Customer Managed Keys?
    (na/3.9)
```

File: docs/integrations/rapid7/index.md [hunk 2]
```diff
@@ -2,22 +2,12 @@
...
 ## Setup
 
-In this section, please provide details about how to set up the integration with
-JupiterOne. This may require provisioning some resources on the provider's side
-(perhaps a role, app, or api key) and passing information over to JupiterOne.
+JupiterOne provides a managed integration for Rapid7 InsightVM. The integration
+connects directly to Rapid7 InsightVM API to obtain configuration metadata and
+analyze resource relationships.
 
 ## Data Model
```
Comment (alee-j1) [inline docs/integrations/rapid7/index.md:9]: This could also be removed since there's another "Data Model" heading below

**Review by alee-j1 — CHANGES_REQUESTED** [review]
just one comment for the rapid 7 docs page for the duplicate "Data Model" heading

**Review by henrymgarrett — APPROVED** [review]

File: release-notes/2021-66.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+- Redacting `activation_code`, `encoded_activation_code`, `uninstall_code` from
+  sensor entities
+
+### GitHub
+
+- Added `createdOn` and `updatedOn` to ingested `github_repo` entities
+
+### Google Cloud
+
+- Migrated integration to latest SDK and infrastructure for more reliable execution
```
Comment (austinkelleher) [inline release-notes/2021-66.md:69]: This looks like copy/paste. Google Cloud has always been on the new SDK. However, GCP did add the following:

```
- Support for ingesting the following **new** resources

- New relationships

  - GKE
    - `google_container_node_pool` **HAS** `google_compute_instance_group`

- New properties added to resources:
  - GKE
    - `google_container_node_pool`
      - `serviceAccount`
      - `bootDiskKmsKey`
  - `google_container_cluster`
    - `workloadIdentity`
```

**Review by austinkelleher — CHANGES_REQUESTED** [review]

File: release-notes/2021-66.md (hunk 3 above)
Comment (aiwilliams) [reply release-notes/2021-66.md:69]: It was meant to capture the move to ECS to allow the integration to complete. That was done this past sprint in https://bitbucket.org/jupiterone/jupiter-integration-google-cloud/pull-requests/26.

Thanks for catching the missing new resources!

**Comment by erkangz** [comment]
Good to go

**Pull request merged**